packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -92,49 +56,17 @@ export async function prepareBuilderDir() {
   const builderDir = join(await cacheDirPromise, 'builders');
   await mkdirp(builderDir);
 
-  // Extract the bundled `builders.tar.gz` file, if necessary
-  const bundledTarballPath = join(__dirname, '../../../assets/builders.tar.gz');
-
-  const existingPackageJson =
-    (await readFileOrNull(join(builderDir, 'package.json'), 'utf8')) || '{}';
-  const { dependencies = {} } = JSON.parse(existingPackageJson);
-
-  if (!hasBundledBuilders(dependencies)) {
-    const extractor = extract(builderDir);
-    await pipe(
-      createReadStream(bundledTarballPath),
-      createGunzip(),
-      extractor
-    );
-  }
-
-  return builderDir;
-}
-
-export async function prepareBuilderModulePath() {
-  const [builderDir, builderContents] = await Promise.all([
-    builderDirPromise,
-    readFile(join(__dirname, 'builder-worker.js')),
-  ]);
-  let needsWrite = false;
-  const builderSha = getSha(builderContents);
-  const cachedBuilderPath = join(builderDir, 'builder.js');
-
-  const cachedBuilderContents = await readFileOrNull(cachedBuilderPath);
-  if (cachedBuilderContents) {
-    const cachedBuilderSha = getSha(cachedBuilderContents);
-    if (builderSha !== cachedBuilderSha) {
-      needsWrite = true;
+  // Create an empty `package.json` file, only if one does not already exist
+  try {
+    const buildersPkg = join(builderDir, 'package.json');
+    await writeJSON(buildersPkg, { private: true }, { flag: 'wx' });
```

Comment:
Do we need to create a `package.json`?

I thought the purpose of this PR was to read directly from `node_modules` instead of installing?

Reply:
Oh I realize we need this for community runtimes like `now-rust` 👍

Reply:
Right. Only for community builders.